pysat/instruments/methods/testing.py

```diff
@@ -29,3 +76,49 @@ def generate_fake_data(t0, num_array, period=5820, data_range=[0.0, 24.0],
         data = ((t0 + num_array) / period).astype(int)
 
     return data
+
+
+def generate_times(fnames, sat_id, freq='1S'):
+    """Construct list of times for simulated instruments
+
+    Parameters
+    ----------
+    fnames : (list)
+        List of filenames
```

Comment:
Only the first filename is used. The current testing standard is to only have one file per day but if we change that this code will break.

We may need to add a note about that fnames is parsed based upon the standard testing filename to extract the appropriate starting date.

Reply:
Since multi-file support is another issue, I think we can note it here and fix that for pysat 3.0

Reply:
Fair enough